Mathlib/Analysis/LocallyConvex/Bounded.lean

```diff
@@ -170,25 +173,26 @@ theorem isVonNBounded_of_smul_tendsto_zero {ε : ι → 𝕝} {l : Filter ι} [l
     rw [Absorbs] at hVS
     push_neg  at hVS
     rcases hVS _ (norm_pos_iff.mpr <| inv_ne_zero hn) with ⟨a, haε, haS⟩
-    rcases set.not_subset.mp haS with ⟨x, hxS, hx⟩
+    rcases Set.not_subset.mp haS with ⟨x, hxS, hx⟩
     refine' ⟨⟨x, hxS⟩, fun hnx => _⟩
     rw [← Set.mem_inv_smul_set_iff₀ hn] at hnx
     exact hx (hVb.smul_mono haε hnx)
   rcases this.choice with ⟨x, hx⟩
   refine' Filter.frequently_false l (Filter.Eventually.frequently _)
   filter_upwards [hx,
-    (H (coe ∘ x) fun n => (x n).2).Eventually (eventually_mem_set.mpr hV)]using fun n => id
+    (H (_ ∘ x) fun n => (x n).2).eventually (eventually_mem_set.mpr hV)] using fun n => id
 #align bornology.is_vonN_bounded_of_smul_tendsto_zero Bornology.isVonNBounded_of_smul_tendsto_zero
 
 /-- Given any sequence `ε` of scalars which tends to `𝓝[≠] 0`, we have that a set `S` is bounded
   if and only if for any sequence `x : ℕ → S`, `ε • x` tends to 0. This actually works for any
   indexing type `ι`, but in the special case `ι = ℕ` we get the important fact that convergent
   sequences fully characterize bounded sets. -/
-theorem isVonNBounded_iff_smul_tendsto_zero {ε : ι → 𝕝} {l : Filter ι} [l.ne_bot]
+theorem isVonNBounded_iff_smul_tendsto_zero {ε : ι → 𝕝} {l : Filter ι} [l.NeBot]
     (hε : Tendsto ε l (𝓝[≠] 0)) {S : Set E} :
     IsVonNBounded 𝕝 S ↔ ∀ x : ι → E, (∀ n, x n ∈ S) → Tendsto (ε • x) l (𝓝 0) :=
   ⟨fun hS x hxS => hS.smul_tendsto_zero (eventually_of_forall hxS) (le_trans hε nhdsWithin_le_nhds),
-    isVonNBounded_of_smul_tendsto_zero (hε self_mem_nhdsWithin)⟩
+    -- Porting note: was `isVonNBounded_of_smul_tendsto_zero (hε self_mem_nhdsWithin)`
+    by apply isVonNBounded_of_smul_tendsto_zero; exact hε self_mem_nhdsWithin⟩
```

Comment:
This one is a little weird; does `(isVonNBounded_of_smul_tendsto_zero (hε self_mem_nhdsWithin) :)` or `isVonNBounded_of_smul_tendsto_zero  (by exact hε self_mem_nhdsWith)` work?

Reply:
The latter works @eric-wieser; I've just edited that in.